### ISSUE TITLE
machine/esp32: uart needs receive

### ISSUE THREAD

**Issue by deadprogram** [tinygo-org/tinygo#1515]
The ESP32 UART does not seem to have RX connected yet, I think because it also needs interrupts implemented.

**Comment by aykevl** [comment]
Yes, interrupts are going to be interesting and will likely require a fair bit of effort to implement. However, IIRC it has some internal buffers that can be used instead of interrupts.

**Comment by deadprogram** [comment]
We have interrupt support now for ESP32, if someone wanted to take a peek at implementing this :smile_cat:

**Comment by aykevl** [comment]
> We have interrupt support now for ESP32, if someone wanted to take a peek at implementing this 😸

What am I missing? Because I don't remember seeing this addition (and I spent _days_ trying to get it working, it's not easy).

**Comment by deadprogram** [comment]
Oops, I confused that fact that we are now generating the wrappers for the interrupts from the SVD file with actually activating the interrupt controller etc. My bad.
